index-import.cpp

```diff
@@ -35,6 +37,9 @@ static cl::opt<bool> Verbose("verbose",
                              cl::desc("Print path remapping results"));
 static cl::alias VerboseAlias("V", cl::aliasopt(Verbose));
 
+static cl::opt<bool> DisableParallel("disable-parallel",
+                                   cl::desc("Disable any parallel processing"));
```

Comment:
If you have `clang-format`, would you mind running it?

Reply:
Done